### ISSUE TITLE
JSX is not displayed correctly in webpage

### ISSUE THREAD

**Issue by aromalanil** [airbnb/javascript#2301]
## Description
- The JSX example is not correctly displayed in the webpage, there is no problem with the Reame file though

## Screenshot
![image](https://user-images.githubusercontent.com/49222186/95022085-959cb880-0692-11eb-956c-e7df32b5cd6c.png)

## Reason
I think the markdown to HTML convertor is what causing the issue

**Comment by aromalanil** [comment]
Here is the Readme Link: https://github.com/airbnb/javascript/blob/master/react/README.md#quotes

**Comment by khareyash05** [comment]
Is this still open? I would like to work on the same...if its open

**Comment by ljharb** [comment]
@khareyash05 yes, it's open, and there's no linked PR, and it has the "pull request wanted" label, so go for it.

**Comment by JeremyRivera** [comment]
So just editing single and double quotes through out?

**Comment by Blkwidows22** [comment]
Are these codes to like put on a chat site or on somebody's phone or I'm new at this I'm sorry I don't know how this works I'm still trying to figure out where to put these codes

**Comment by JatinSainiOO7** [comment]
Hello!
Can I Work On This Issue.
Thankyou.

**Comment by itsjeremyrivera** [comment]
Sure!Jeremy ***@***.*** / 813-352-1618On Nov 3, 2023, at 01:29, JATIN SAINI ***@***.***> wrote:﻿
Hello!
Can I Work On This Issue.
Thankyou.

**Comment by Robbie-BOIII** [comment]
Kamino cloned this issue to [Robbie-BOIII/test](https://github.com/Robbie-BOIII/test/issues/3)

**Comment by DetrojaRadhey** [comment]
I'm interested in working on this issue. Could you please confirm if it's still open?

**Comment by ljharb** [comment]
@DetrojaRadhey you can see at the top of the issue that it's open, but you can also see that #2451 may fix it. Thus, if you have a contribution, please comment with a branch link on #2451, and I can pull it into the existing PR.

**Comment by DetrojaRadhey** [comment]
@ljharb, I have used HTML character entities to display JSX code in a markdown format without being rendered as HTML.

Review this PR and let me know if any changes needed:
https://github.com/airbnb/javascript/pull/3060

**Comment by ljharb** [comment]
@DetrojaRadhey i did that, yesterday.
